src/template/index.ts

```diff
@@ -71,3 +71,28 @@ export async function renderHeaderFooter(proposalId?: string) {
     footerTemplate: handlebar.compile(htmlFooterTemplate)(null),
   };
 }
+
+export async function renderHeader(proposalId?: string) {
+  const htmlHeaderTemplate = await promises.readFile(
+    join(templatesFolder, 'pdf', 'header.hbs'),
+    'utf-8'
+  );
+
+  const headerData = {
+    logoPath: process.env.HEADER_LOGO_PATH
+      ? process.env.HEADER_LOGO_PATH
+      : join(process.cwd(), './templates/images/ESS.png'),
```

Comment:
Maybe you have already discussed this, but I wonder if it would be better to use a neutral logo in case HEADER_LOGO_PATH is not specified, or a white 1x1 pixel?